tests/core/util/test_jsonify.py

```diff
@@ -0,0 +1,113 @@
+from dataclasses import dataclass
+from typing import List, Optional, Tuple
+
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.util.ints import uint32
+from chia.util.streamable import Streamable, recurse_jsonify, streamable
+
+
+def test_primitives() -> None:
+    @streamable
+    @dataclass(frozen=True)
+    class PrimitivesTest(Streamable):
+        a: uint32
+        b: Optional[str]
+        c: str
+        d: bytes
+        e: bytes32
+
+    t = PrimitivesTest(
+        uint32(123),
+        None,
+        "foobar",
+        b"\0\1\0\1",
+        bytes32(
+            [
+                0,
+                1,
+                2,
+                3,
+                4,
+                5,
+                6,
+                7,
+                8,
+                9,
+                10,
+                11,
+                12,
+                13,
+                14,
+                15,
+                16,
+                17,
+                18,
+                19,
+                20,
+                21,
+                22,
+                23,
+                24,
+                25,
+                26,
+                27,
+                28,
+                29,
+                30,
+                31,
+            ]
+        ),
```

Comment:
This way I don't have to read every line to know what is happening.

```suggestion
        bytes32(range(32)),
```

Reply:
heh, yeah. it didn't look like this before `black` btw :)